stories/object-identifier/object-identifier.stories.js

```diff
@@ -0,0 +1,173 @@
+import '../../dist/object-identifier.css';
+import '../../dist/link.css';
+import '../../dist/table.css';
+
+export default {
+    title: 'Components/Object Identifier',
+    parameters: {
+        description: `
+The object identifier is a short text that represents the key identifier of an object, and can be displayed in various types of containers, tables, panels, popovers and more. The text wraps if there is insufficient horizontal space.
+
+The object identifier should be easily read by the users, preferably displaying text instead of an ID. If it necessary to distinguish objects by their IDs, then it should appear as descriptive text below the title text.
+
+The object number comprises:
+
+- A title text
+- An optional descriptive text
+- An active state (if desired) which resembles a link that can trigger an event
+
+##Usage
+Use the object identifier if:
+
+- You want to indicate the key identifier on an object.
+
+Do not use the object identifier if:
+
```

Comment:
And bold here

Reply:
Basically wherever there is a ":" at the end of the sentence is the general rule

Reply:
fixed